benchexec/tools/WitForJBMC.py

```diff
@@ -0,0 +1,32 @@
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
```

Comment:
Please add some documentation about the tool as [suggested here](https://github.com/sosy-lab/benchexec/commit/bd82323567b2c8ebade40b8778460de1dda53308#diff-564acd0f58dd345e8d161cb791d6af53), i.e., at least name and URL. An example is [here](https://github.com/sosy-lab/benchexec/blob/bd82323567b2c8ebade40b8778460de1dda53308/benchexec/tools/cpachecker.py).